content/v2.0/reference/api/influxdb-1x/dbrp.md

```diff
@@ -0,0 +1,155 @@
+2. Check to see if a bucket matching the naming convention, `<database>/<retention-policy>`, exists.
+  - If a bucket with that name exists, add the DBRP mapping to the existing bucket
+     and write the data to the bucket.
+  - If no bucket with that name exists, then:
+3. Create a new bucket using the naming convention `<database>/<retention-policy>`,
+   add the DBRP mapping to the new bucket, and write the data to the bucket.
+
+##### If no retention policy is specified:
+1. Check for the specified database in the DBRP mapping service.
+  - If the database exists, get the default retention policy for the specified database, and then:
+  - If the database doesn't exists, add the database, set its default retention policy as `autogen`, and then:
+2. Continue the [normal DBRP mapping behavior](#when-writing-data).
+
+### When querying data
+When querying data from InfluxDB 2.0 using the [`/query` compatibility endpoint](/v2.0/reference/api/influxdb-1x/query/),
+the DBRP mapping service uses the specified database and retention policy to:
+
+Check for a DBRP mapping that matches the database and retention policy.
+
+- If a DBRP mapping exists, query data from the mapped bucket.
+- If no mapping exists, return an error.
+
+##### If no retention policy is specified:
+1. Check for the specified database in the DBRP mapping service.
+  - If the database doesn't exists, return an error.
+  - If database exists, get the default retention policy for the specified database, and then:
+2. Continue the [normal DBRP mapping behavior](#when-querying-data).
+
+## Automatically create DBRP mappings
+
+- [Create a new bucket and DBRP mapping](#create-a-new-bucket-and-dbrp-mapping)
+- [Automatically map to existing buckets](#automatically-map-to-existing-buckets)
+
+{{% warn %}}
+Only the `/write` compatibility endpoint automatically creates DBRP mappings.
+The `/query` compatibility endpoint will return an error if querying a database
+and retention policy combination that is not mapped to a bucket.
+{{% /warn %}}
+
+### Create a new bucket and DBRP mapping
+When using the `/write` compatibility endpoint to write data to InfluxDB 2.0
+and no DBRP mapping exists for the specified database and retention policy and
+no bucket exists that matches the `<database>/<retention-policy>` naming convention,
+InfluxDB **creates a new bucket** using the `<database>/<retention-policy>` naming convention.
+
+**For example:**
+
+```sh
+curl -XPOST http://localhost:9999/write?db=mydb&rp=myrp \
+  -H "Authorization: Token YourAuthToken" \
+  --data-binary "measurement,host=host1 field1=2i,field2=2.0 1577836800000000000"
+```
+
+If no DBRP mapping exists for the `mydb` and `myrp` combination **and** no bucket
+exists with the name `mydb/myrp`, InfluxDB automatically creates a new bucket named
+`mydb/myrp` and writes data to the new bucket.
+
+_See [DBRP behavior when writing data](#when-writing-data)._
+
+### Automatically map to existing buckets
```

Comment:
ditto comment above. 

Would folks look for how to "Automatically map to existing buckets" in a context other than "Writing 1.x data to 2.0"?

Reply:
Dropped this section